managed/cmd/pmm-managed/main.go

```diff
@@ -663,10 +663,28 @@ func main() {
 	versionServiceAPIURLF := kingpin.Flag("version-service-api-url", "Version Service API URL").
 		Default("https://check.percona.com/versions/v1").Envar("PERCONA_TEST_VERSION_SERVICE_URL").String()
 
-	postgresAddrF := kingpin.Flag("postgres-addr", "PostgreSQL address").Default("127.0.0.1:5432").String()
-	postgresDBNameF := kingpin.Flag("postgres-name", "PostgreSQL database name").Required().String()
-	postgresDBUsernameF := kingpin.Flag("postgres-username", "PostgreSQL database username").Default("pmm-managed").String()
-	postgresDBPasswordF := kingpin.Flag("postgres-password", "PostgreSQL database password").Default("pmm-managed").String()
+	postgresAddrF := kingpin.Flag("postgres-addr", "PostgreSQL address").
```

Comment:
What do you think about using Postgres connection string instead of separating individual pieces of information?

Reply:
This can be discussed with leads. In my opinion, it's better not to.
1. With environment variables approach, there is no need to rewrite the logic of connection functions.
2. For the client this will be obvious, since we initially designate the minimum set of values necessary for us. The rest we configure ourselves for our needs.
3. Otherwise, we will need to have something like a black and white list of parameters that will work for us. And most likely for different versions of databases.
4. And if these parameters are not suitable, we will need to report it, otherwise the process for the end user will be non-obvious and most likely inconvenient.

Reply:
The connection string provides more flexibility in terms of specifying connection parameters, such as supporting [multiple hosts](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-MULTIPLE-HOSTS).

I think it's worth considering using this approach so we don't limit our users from using common connection options.

Reply:
we can't use just the connection string, because grafana doesn't support connection strings, so we will have to add additional logic to parse the connection string in ansible.

Reply:
then what about services that do not support multiple hosts in this case?

Reply:
my comment is outdated. Yes, so as you say that grafana doesn't support connection string...  I also meant it.